python/external_models/requirement.txt

```diff
@@ -0,0 +1,6 @@
+loguru
+transformers
+anthropic
+openai
+google
+vllm
```

Comment:
1. Let's rename this file to be `requirements.txt`, plural.
2. Let's move it to the outer level, right inside `python/`. Plus, there is a README in `python/`, and we can make the installation of pip packages simpler with a `pip install -r requirements.txt`.
3. Is this list comprehensive? Let us assume that the user starts with no python packages and relies solely on this `requirements.txt` to install necessary dependencies. You may create an empty conda environment to test it out.
4. Is there any restriction introduced by such required packages? E.g. I vaguely remember `vllm` may not work nicely with Mac. Do we have such constraints? If so we should either try to get rid of them or at least mention in the readme.

Reply:
I'm not sure, because vllm is always used for GPU servers. I guess for mac huggingface should be the best solution.